### PR TITLE
Add git branch and commit when log is initialized to help debug

### PR DESCRIPTION
**Pull request by ademar111190**
How the log looks like:

```
[Logger] {FINE} (2023-10-11T12:17:10.440217Z) : Logging initialized, app build on main at commit 06b48d40ed9e7093efb0640ad5846ab61e0e2038
```

No QA needed

### CODE REVIEW

**Comment by JssDWt** [comment]
Looks like a lot of files are being added to the build just to show the git commit in the logs? What is the size of these .git/** files currently? Would it be possible to simply get the git commit hash during build and put it in a file?

**Comment by ademar111190** [comment]
Adding this log would help a lot in cases like this: https://github.com/breez/c-breez/issues/671#issuecomment-1759552864

**Review by ubbabeck — APPROVED** [review]
tACK

File: lib/logger.dart [hunk 1]
```diff
@@ -60,6 +61,12 @@ class BreezLogger {
       final exception = details.exceptionAsString();
       _log.severe("$exception -- $name", details, details.stack);
     };
+
+    GitInfo.get().then((it) {
+      _log.fine("Logging initialized, app build on ${it.branch} at commit ${it.hash}");
```
[the remark this answers is not in the document]
Comment (ademar111190) [reply lib/logger.dart:66]: true, thanks @erdemyerebasmaz

**Pull request merged**